bin/get-github-oauth-token

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+
+var Github = require('github'),
+    read   = require('read');
+
+read({
+   prompt: "Github Username:"
+}, function(error, response) {
+   getPassword({username: response});
+})
+
+function getPassword(auth) {
+   read({
+      silent: true,
+      prompt: "Github Password:"
+   }, function(error, response) {
+      auth.password = response;
+      getOTP(auth);
+   })
+}
+
+function getOTP(auth) {
+   read({
+      prompt: "2-Factor Auth Code (leave blank for none):"
```

Comment:
How does the user get their "2-Factor Auth Code"? Wouldn't we need to call `github.authenticate()` before they get a text?